src/main/scala/com/databricks/labs/overwatch/pipeline/SilverTransforms.scala

```diff
@@ -1020,13 +1023,44 @@ trait SilverTransforms extends SparkSessionWrapper {
 
     val invalidEventChain = lead('runningSwitch, 1).over(stateUnboundW).isNotNull && lead('runningSwitch, 1)
       .over(stateUnboundW) === lead('previousSwitch, 1).over(stateUnboundW)
-    val clusterEventsBaseline = clusterEventsDF
+
+
+    val clusterEventsDF1 = clusterEventsDF
```

Comment:
I know these inner variables are hard to name but let's find something more descriptive than xyzDF1. Longer variable names are fine

Reply:
Done